src/web/mjs/engine/Settings.mjs

```diff
@@ -286,6 +283,7 @@ export default class Settings {
         Engine.Storage.saveConfig( 'settings', data, 2 )
             .then( () => {
                 document.dispatchEvent( new CustomEvent( EventListener.onSettingsChanged, { detail: this } ) );
+                document.dispatchEvent( new CustomEvent( EventListener.onSettingsSaved, { detail: this } ) );
```

Comment:
I had to add a new event listener here. 
the `onSettingsChanged` is being created twice, once here `line 285` and above `line 255`.

`Line 255` is being called once hakuneko is started.
`(Line 233 in current master)`
https://github.com/manga-download/hakuneko/blob/9a52af9cf2605bdb922a61b323f3b06ddb968220/src/web/mjs/engine/Settings.mjs#L233
problem with that is, if I bind the `savebookmark` function to `onSettingsChanged`
it will save a empty bookmark, since its not loaded yet.